tests-functional/conftest.py

```diff
@@ -52,6 +60,24 @@ class Option:
 def pytest_configure(config):
     global option
     option = config.option
-    if type(option.status_backend_urls) is str:
-        option.status_backend_urls = option.status_backend_urls.split(",")
+
+    executor_number = int(os.getenv('EXECUTOR_NUMBER', 5))
+    base_port = 7000
+    range_size = 100
+
+    start_port = base_port + (executor_number * range_size)
+
+    option.status_backend_port_range = list(range(start_port, start_port + range_size - 1))
+    option.status_backend_containers = []
+
     option.base_dir = os.path.dirname(os.path.abspath(__file__))
+
+def pytest_unconfigure():
+    docker_client = docker.from_env()
+    for container_id in option.status_backend_containers:
+        try:
+            container = docker_client.containers.get(container_id)
+            container.stop(timeout=30)
+            container.remove()
```

Comment:
If I understand correctly, `pytest_unconfigure` is called when ALL tests have finished.
But maybe we can call `container.stop` at each test end? We could then free the used port as well.

Reply:
we have many tests cases that are using single instance of status backend for all tests inside, we do not have test == container, yet

Reply:
as result to stop at each test end we need to start at each test :) possible but I would rather do it outside this PR and review if we really need that many instances

Reply:
Sure, let's do it in a separate PR 👍